### DISCUSSION TITLE
I'm getting syntax error vendor/spatie/crawler/src/Crawler.php:29

### DISCUSSION THREAD

**Discussion by hassanshahzadaheer** [spatie/laravel-sitemap#493] [General]
ParseError
syntax error
http://petservice.test/sitemap.xml

I have updated the spatie/crawler
`composer update spatie/crawler`
and downgrading 
`composer require spatie/crawler:^5.0`

and run the **http://petservice.test/sitemap.xml** still have the same issue

please guide me how to resolve this.

**Comment by freekmurze** [comment]
The tests prove that we don't have any syntax errors.

Probably you're running this on a PHP version that is not supported by the package.

**Reply by hassanshahzadaheer** [reply]
Here is my status on system
name     : spatie/laravel-sitemap
descrip. : Create and generate sitemaps with ease
keywords : laravel-sitemap, spatie
versions : * 5.9.2
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
homepage : https://github.com/spatie/laravel-sitemap
source   : [git] https://github.com/spatie/laravel-sitemap.git df5c3db511e08a8e64a3d7e28613ab018a99e95d
dist     : [zip] https://api.github.com/repos/spatie/laravel-sitemap/zipball/df5c3db511e08a8e64a3d7e28613ab018a99e95d df5c3db511e08a8e64a3d7e28613ab018a99e95d
path     : /Applications/XAMPP/xamppfiles/htdocs/application/vendor/spatie/laravel-sitemap
names    : spatie/laravel-sitemap

support
issues : https://github.com/spatie/laravel-sitemap/issues
source : https://github.com/spatie/laravel-sitemap/tree/5.9.2

autoload
psr-4
Spatie\Sitemap\ => src

requires
guzzlehttp/guzzle ^7.2
illuminate/support ^8.0
nesbot/carbon ^2.0
php ^7.4|^8.0
spatie/crawler ^5.0|^6.0
symfony/dom-crawler ^5.1.14

requires (dev)
mockery/mockery ^1.3.3
orchestra/testbench ^6.0
phpunit/phpunit ^9.3
spatie/phpunit-snapshot-assertions ^4.0
spatie/temporary-directory ^1.1

 "php": "^7.3|^8.0",
laravel/framework": "^8.0"
